### PR TITLE
Translate the string to supported CQL temporal operator

### PR DESCRIPTION
**Pull request by vietnguyengit**

### CODE REVIEW

File: server/src/main/java/au/org/aodn/ogcapi/server/core/service/OGCApiService.java [hunk 1]
```diff
@@ -78,13 +78,16 @@ public String processDatetimeParameter(String datetime, String filter) {
...
+                operator = "before";
+            } else if (datetime.contains("/..")) {
+                operator = "after";
+            } else if (datetime.contains("/") && !datetime.contains("..")) {
                 operator = "during";
-                filter = String.format("temporal %s %s", operator, datetime);
             } else {
-                // TODO: this where problem is, should be after or before if input is a single timestamp instant?
-                operator = "after";
+                operator = "tequals";
```
Comment (vietnguyengit) [inline server/src/main/java/au/org/aodn/ogcapi/server/core/service/OGCApiService.java:88]: not incorrect spelling, it's `tequals`

File: server/src/main/java/au/org/aodn/ogcapi/server/core/service/OGCApiService.java [hunk 2]
```diff
@@ -78,13 +78,16 @@ public String processDatetimeParameter(String datetime, String filter) {
 
         // for now, assumption is that temporal is the only filter
         if (filter == null) {
-            if (datetime.contains("/")) {
+            if (datetime.contains("../")) {
```
Comment (utas-raymondng) [inline server/src/main/java/au/org/aodn/ogcapi/server/core/service/OGCApiService.java:81]: should be datetime.startsWith("../") || datetime.startsWith("/"), 
same idea applies to the reset

**Review by utas-raymondng — CHANGES_REQUESTED** [review]

File: server/src/main/java/au/org/aodn/ogcapi/server/core/service/OGCApiService.java (hunk 2 above)
Comment (utas-raymondng) [reply server/src/main/java/au/org/aodn/ogcapi/server/core/service/OGCApiService.java:81]: for endsWith ('/') || endsWith ("/..")

File: server/src/main/java/au/org/aodn/ogcapi/server/core/service/OGCApiService.java [hunk 3]
```diff
@@ -78,13 +78,16 @@ public String processDatetimeParameter(String datetime, String filter) {
 
         // for now, assumption is that temporal is the only filter
         if (filter == null) {
-            if (datetime.contains("/")) {
+            if (datetime.startsWith("../") || datetime.startsWith("/")) {
+                operator = "before";
+            } else if (datetime.endsWith("/..") || datetime.endsWith("/")) {
+                operator = "after";
+            } else if (datetime.contains("/") && !datetime.contains("..")) {
                 operator = "during";
```
Comment (vietnguyengit) [inline server/src/main/java/au/org/aodn/ogcapi/server/core/service/OGCApiService.java:86]: oh still need to update this @utas-raymondng

**Review by utas-raymondng — APPROVED** [review]
LGTM

**Pull request merged**